app/workers/github/create_issue_job.rb

```diff
@@ -30,6 +32,9 @@ def perform(feedback)
         create_response = Github::GithubService.create_issue(feedback)
         FeedbackSubmissionMailer.build(feedback, create_response&.html_url, create_response&.number).deliver_now
       end
+    rescue Exception => e
+      StatsD.increment(metric_name(self, 'rate_limited')) if e.class.name == 'Sidekiq::Limiter::OverLimit'
```

Comment:
:+1: for comparing the class as a string so as not to run into runtime exceptions.